flash_attn/flash_attn_triton.py

```diff
@@ -201,7 +201,7 @@ def _fwd_kernel(
         # -- update statistics
         m_i = m_ij
         l_i_new = tl.exp(lse_i - m_ij) + l_ij
-        lse_i = m_ij + tl.log(l_i_new)
+        lse_ii = m_ij + tl.log(l_i_new)
```

Comment:
WIP debugging: this is real error location for the error message

```
loc(callsite("/mnt/xarfuse/uid-169890/249f3187-seed-nspid4026531836_cgpid3845545-ns-4026531840/triton/language/core.py":1398:21 at "/mnt/xarfuse/uid-169890/249f3187-seed-nspid4026531836_cgpid3845545-ns-4026531840/flash_attn/flash_attn_triton.py":165:61)): error: 'triton_gpu.cmpf' op requires the same encoding for all operands
```

Reply:
@jianyuh Is this fixed on triton nightly?

Reply:
@Skylion007 No. Triton nightly has not trans_b.